### PR TITLE
Units trajectories model

### PR DESCRIPTION
**Pull request by RichMicke**
The units_trajectory model was created

### CODE REVIEW

File: db/migrate/20240503213904_create_units_trajectories_models.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateUnitsTrajectoriesModels < ActiveRecord::Migration[7.1]
+  def change
+    create_table :units_trajectories_models do |t|
+
+      t.timestamps
+    end
+  end
+end
```
Comment (mtopete1996) [inline db/migrate/20240503213904_create_units_trajectories_models.rb]: ❓

File: lib/trajectories/route_22.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (mtopete1996) [inline lib/trajectories/route_22.json]: ❓

**Review by mtopete1996 — APPROVED** [review]
Extra files ❓ ❓

**Review by mtopete1996 — CHANGES_REQUESTED** [review]
extra files ❓❓

File: test/fixtures/units_trajectories_models.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (jesusMava) [inline test/fixtures/units_trajectories_models.yml:1]: are we going to use fixtures?

File: db/migrate/20240503213904_create_units_trajectories_models.rb (hunk 1 above)
Comment (jesusMava) [reply db/migrate/20240503213904_create_units_trajectories_models.rb]: same question that @mtopete1996 , what are you trying to do here?

File: app/models/units_trajectories_model.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class UnitsTrajectoriesModel < ApplicationRecord
+    class UnitsTrajectory < ApplicationRecord
+    belongs_to :unit
```
Comment (jesusMava) [inline app/models/units_trajectories_model.rb]: a model inside other model?

**Review by jesusMava — CHANGES_REQUESTED** [review]

File: app/models/units_trajectories_model.rb (hunk 4 above)
Comment (mtopete1996) [reply app/models/units_trajectories_model.rb]: Why is this called Model at the end?

File: app/models/city_stop.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+class CityStop < ApplicationRecord
```
Comment (mtopete1996) [inline app/models/city_stop.rb]: remove this

**Review by mtopete1996 — APPROVED** [review]
LGTM :)

File: app/models/city_stop.rb [hunk 6]
```diff
@@ -2,7 +2,5 @@ class CityStop < ApplicationRecord
     belongs_to :stop
     belongs_to :city
       
-    validates :stop_id, :city_id, presence: true
-
-    end  
+    validates :stop_id, :city_id, presence: true
```
Comment (jesusMava) [inline app/models/city_stop.rb]: 🤔 good job, but i dont get it, what did it change?

**Review by jesusMava — APPROVED** [review]

**Review by mtopete1996 — APPROVED** [review]
LGTM :)

**Pull request merged**